main.go

```diff
@@ -36,12 +43,17 @@ func main() {
 
 	log.Printf("OperationalMode: %s\n", config.WatchdogMode(watchdogConfig.OperationalMode))
 
-	if err := lock(); err != nil {
-		log.Panic(err.Error())
+	path, lockErr := lock()
+
+	if lockErr != nil {
+		log.Panicf("Cannot write %s. To disable lock-file set env suppress_lock=true.\n Error: %s.\n", path, lockErr.Error())
```

Comment:
Have you tested this @BurtonR ? I can't see any code that handles a `suppress_lock` environmental variable.

Reply:
This was a relic of the copy/paste from the original watchdog. I'll need to double check we can still disable the lock file here and test it. I'll need to update that panic message regardless